diff -ruN SoundTouch-1.2.1-old/source/SoundTouch/Makefile.am SoundTouch-1.2.1/source/SoundTouch/Makefile.am
--- SoundTouch-1.2.1-old/source/SoundTouch/Makefile.am	2003-12-27 13:04:38.000000000 +0100
+++ SoundTouch-1.2.1/source/SoundTouch/Makefile.am	2004-08-10 23:40:32.000000000 +0200
@@ -39,4 +39,5 @@
 
 # other linking flags to add
 #libSoundTouch_la_LIBADD=
+libSoundTouch_la_LDFLAGS= -version-info 1:0:0
 
diff -ruN SoundTouch-1.2.1-old/source/example/bpm/Makefile.am SoundTouch-1.2.1/source/example/bpm/Makefile.am
--- SoundTouch-1.2.1-old/source/example/bpm/Makefile.am	2003-12-27 13:08:12.000000000 +0100
+++ SoundTouch-1.2.1/source/example/bpm/Makefile.am	2004-08-10 23:40:46.000000000 +0200
@@ -37,4 +37,5 @@
 
 # other linking flags to add
 #libBPM_la_LIBADD=
+libBPM_la_LDFLAGS= -version-info 1:0:0
 
