diff -Nur soundtouch-1.3.0/build-tree/SoundTouch-1.3.0/include/Makefile.in soundtouch-1.3.0.new/build-tree/SoundTouch-1.3.0/include/Makefile.in
--- SoundTouch-1.3.0/include/Makefile.in	2005-02-06 07:21:59.000000000 -0500
+++ SoundTouch-1.3.0/include/Makefile.in	2008-03-10 14:38:03.000000000 -0400
@@ -304,7 +304,7 @@
 installcheck: installcheck-am
 install-strip:
 	$(MAKE) $(AM_MAKEFLAGS) INSTALL_PROGRAM="$(INSTALL_STRIP_PROGRAM)" \
-	  install_sh_PROGRAM="$(INSTALL_STRIP_PROGRAM)" INSTALL_STRIP_FLAG=-s \
+	  install_sh_PROGRAM="$(INSTALL_STRIP_PROGRAM)" INSTALL_STRIP_FLAG= \
 	  `test -z '$(STRIP)' || \
 	    echo "INSTALL_PROGRAM_ENV=STRIPPROG='$(STRIP)'"` install
 mostlyclean-generic:
diff -Nur soundtouch-1.3.0/build-tree/SoundTouch-1.3.0/Makefile.in soundtouch-1.3.0.new/build-tree/SoundTouch-1.3.0/Makefile.in
--- SoundTouch-1.3.0/Makefile.in	2005-02-06 07:21:59.000000000 -0500
+++ SoundTouch-1.3.0/Makefile.in	2008-03-10 14:38:03.000000000 -0400
@@ -570,7 +570,7 @@
 installcheck: installcheck-recursive
 install-strip:
 	$(MAKE) $(AM_MAKEFLAGS) INSTALL_PROGRAM="$(INSTALL_STRIP_PROGRAM)" \
-	  install_sh_PROGRAM="$(INSTALL_STRIP_PROGRAM)" INSTALL_STRIP_FLAG=-s \
+	  install_sh_PROGRAM="$(INSTALL_STRIP_PROGRAM)" INSTALL_STRIP_FLAG= \
 	  `test -z '$(STRIP)' || \
 	    echo "INSTALL_PROGRAM_ENV=STRIPPROG='$(STRIP)'"` install
 mostlyclean-generic:
diff -Nur soundtouch-1.3.0/build-tree/SoundTouch-1.3.0/source/example/bpm/Makefile.in soundtouch-1.3.0.new/build-tree/SoundTouch-1.3.0/source/example/bpm/Makefile.in
--- SoundTouch-1.3.0/source/example/bpm/Makefile.in	2005-02-06 07:22:00.000000000 -0500
+++ SoundTouch-1.3.0/source/example/bpm/Makefile.in	2008-03-10 14:38:03.000000000 -0400
@@ -378,7 +378,7 @@
 installcheck: installcheck-am
 install-strip:
 	$(MAKE) $(AM_MAKEFLAGS) INSTALL_PROGRAM="$(INSTALL_STRIP_PROGRAM)" \
-	  install_sh_PROGRAM="$(INSTALL_STRIP_PROGRAM)" INSTALL_STRIP_FLAG=-s \
+	  install_sh_PROGRAM="$(INSTALL_STRIP_PROGRAM)" INSTALL_STRIP_FLAG= \
 	  `test -z '$(STRIP)' || \
 	    echo "INSTALL_PROGRAM_ENV=STRIPPROG='$(STRIP)'"` install
 mostlyclean-generic:
diff -Nur soundtouch-1.3.0/build-tree/SoundTouch-1.3.0/source/example/Makefile.in soundtouch-1.3.0.new/build-tree/SoundTouch-1.3.0/source/example/Makefile.in
--- SoundTouch-1.3.0/source/example/Makefile.in	2005-02-06 07:22:00.000000000 -0500
+++ SoundTouch-1.3.0/source/example/Makefile.in	2008-03-10 14:38:03.000000000 -0400
@@ -371,7 +371,7 @@
 installcheck: installcheck-recursive
 install-strip:
 	$(MAKE) $(AM_MAKEFLAGS) INSTALL_PROGRAM="$(INSTALL_STRIP_PROGRAM)" \
-	  install_sh_PROGRAM="$(INSTALL_STRIP_PROGRAM)" INSTALL_STRIP_FLAG=-s \
+	  install_sh_PROGRAM="$(INSTALL_STRIP_PROGRAM)" INSTALL_STRIP_FLAG= \
 	  `test -z '$(STRIP)' || \
 	    echo "INSTALL_PROGRAM_ENV=STRIPPROG='$(STRIP)'"` install
 mostlyclean-generic:
diff -Nur soundtouch-1.3.0/build-tree/SoundTouch-1.3.0/source/example/SoundStretch/Makefile.am soundtouch-1.3.0.new/build-tree/SoundTouch-1.3.0/source/example/SoundStretch/Makefile.am
--- SoundTouch-1.3.0/source/example/SoundStretch/Makefile.am	2008-03-10 14:38:02.000000000 -0400
+++ SoundTouch-1.3.0/source/example/SoundStretch/Makefile.am	2008-03-10 14:38:22.000000000 -0400
@@ -44,7 +44,7 @@
 soundstretch_LDADD=../../SoundTouch/libSoundTouch.la ../bpm/libBPM.la -lm
 
 ## linker flags
-soundstretch_LDFLAGS=-s
+soundstretch_LDFLAGS=
 
 ## additional compiler flags
 soundstretch_CXXFLAGS=-O2 -I../bpm
diff -Nur soundtouch-1.3.0/build-tree/SoundTouch-1.3.0/source/example/SoundStretch/Makefile.in soundtouch-1.3.0.new/build-tree/SoundTouch-1.3.0/source/example/SoundStretch/Makefile.in
--- SoundTouch-1.3.0/source/example/SoundStretch/Makefile.in	2005-02-06 07:22:00.000000000 -0500
+++ SoundTouch-1.3.0/source/example/SoundStretch/Makefile.in	2008-03-10 14:38:03.000000000 -0400
@@ -182,7 +182,7 @@
 EXTRA_DIST = soundstretch.dsp soundstretch.dsw
 soundstretch_SOURCES = main.cpp RunParameters.cpp WavFile.cpp 
 soundstretch_LDADD = ../../SoundTouch/libSoundTouch.la ../bpm/libBPM.la -lm
-soundstretch_LDFLAGS = -s
+soundstretch_LDFLAGS = 
 soundstretch_CXXFLAGS = -O3 -I../bpm
 all: all-am
 
@@ -457,7 +457,7 @@
 installcheck: installcheck-am
 install-strip:
 	$(MAKE) $(AM_MAKEFLAGS) INSTALL_PROGRAM="$(INSTALL_STRIP_PROGRAM)" \
-	  install_sh_PROGRAM="$(INSTALL_STRIP_PROGRAM)" INSTALL_STRIP_FLAG=-s \
+	  install_sh_PROGRAM="$(INSTALL_STRIP_PROGRAM)" INSTALL_STRIP_FLAG= \
 	  `test -z '$(STRIP)' || \
 	    echo "INSTALL_PROGRAM_ENV=STRIPPROG='$(STRIP)'"` install
 mostlyclean-generic:
diff -Nur soundtouch-1.3.0/build-tree/SoundTouch-1.3.0/source/Makefile.in soundtouch-1.3.0.new/build-tree/SoundTouch-1.3.0/source/Makefile.in
--- SoundTouch-1.3.0/source/Makefile.in	2005-02-06 07:21:59.000000000 -0500
+++ SoundTouch-1.3.0/source/Makefile.in	2008-03-10 14:38:03.000000000 -0400
@@ -371,7 +371,7 @@
 installcheck: installcheck-recursive
 install-strip:
 	$(MAKE) $(AM_MAKEFLAGS) INSTALL_PROGRAM="$(INSTALL_STRIP_PROGRAM)" \
-	  install_sh_PROGRAM="$(INSTALL_STRIP_PROGRAM)" INSTALL_STRIP_FLAG=-s \
+	  install_sh_PROGRAM="$(INSTALL_STRIP_PROGRAM)" INSTALL_STRIP_FLAG= \
 	  `test -z '$(STRIP)' || \
 	    echo "INSTALL_PROGRAM_ENV=STRIPPROG='$(STRIP)'"` install
 mostlyclean-generic:
diff -Nur soundtouch-1.3.0/build-tree/SoundTouch-1.3.0/source/SoundTouch/Makefile.in soundtouch-1.3.0.new/build-tree/SoundTouch-1.3.0/source/SoundTouch/Makefile.in
--- SoundTouch-1.3.0/source/SoundTouch/Makefile.in	2005-02-06 07:21:59.000000000 -0500
+++ SoundTouch-1.3.0/source/SoundTouch/Makefile.in	2008-03-10 14:38:03.000000000 -0400
@@ -394,7 +394,7 @@
 installcheck: installcheck-am
 install-strip:
 	$(MAKE) $(AM_MAKEFLAGS) INSTALL_PROGRAM="$(INSTALL_STRIP_PROGRAM)" \
-	  install_sh_PROGRAM="$(INSTALL_STRIP_PROGRAM)" INSTALL_STRIP_FLAG=-s \
+	  install_sh_PROGRAM="$(INSTALL_STRIP_PROGRAM)" INSTALL_STRIP_FLAG= \
 	  `test -z '$(STRIP)' || \
 	    echo "INSTALL_PROGRAM_ENV=STRIPPROG='$(STRIP)'"` install
 mostlyclean-generic:
